### ISSUE TITLE
Incorporate work done on INCF in extension proposal

### ISSUE THREAD

**Issue by Gilles86** [bids-standard/bep001#7]
A group of people have done some great work on INCF and we talked about it during a BIDS meeting.

See here for the original document:
https://docs.google.com/document/d/1HfCjPKCXqTckB0OPs-Py06yY2gPYaFGc0sFTPB74fEA/edit#

I think there are some great suggestions, such as using the suffix _MPM for MPM acquisitions and the acq-field to indicate which acquisition is mostly weighted like what.

There are also some nice examples for VFA, MT, B1...

I think it would be good to include them in the current draft. Any opinions on this? Anyone wants to make a commit for this?

See also:
https://docs.google.com/spreadsheets/d/1awzGVflxXiWtzeLVOH5XI8Bar2ND9759yZxsK3stCzc/edit#gid=0

**Comment by agahkarakuzu** [comment]
Thank you @Gilles86 for reading the document. Happy that suggestions sounded reasonable, I put some effort into coming up with a naming convention that can work for everyone and is virtually scalable. I am aware that the list provided in the spreadsheet can be lengthened almost indefinitely. This is the reason why I tried to include methods that can be carried out using stock sequences.

Currently, most of the qMRI vs BIDS discussion is focusing on the terminological accuracy/consistency/quality of certain key tags in the sidecar jsons. I indeed agree that reaching a consensus regarding these issues is principally important to achieve a sustainable standard. 

However, practically, above concerns should **not** prevent suggestions in the INCF document to be included in the current BEP draft. As can be seen in [the spreadsheet](https://docs.google.com/spreadsheets/d/1awzGVflxXiWtzeLVOH5XI8Bar2ND9759yZxsK3stCzc/edit#gid=0), adding them does not bring lots of new key tags to be dealt with, as varying parameters scoped by MPM and the MP2RAGE already cover 4/5 of all varying parameters. I mean 20% of the methods listed in the spreadsheet covers 80% of the varying fields, sounds Pareto efficient. 😄
